### PR TITLE
Rename `sfdc` to `salesforce`

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description
Rename `sfdc` to `salesforce`
Changes are in the documentation,  client modules, Java files etc.

One line release note: 
- Rename `sfdc` to `salesforce`

## Type of change
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**Test Configuration**:
* Ballerina Version: slbeta6
* Operating System: Ubuntu
* Java SDK: Java11

# Checklist:

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by sachinira — COMMENTED** [review]
Check whether the PRs should be for `master` branch or `slbeta6` or both branches with @abeykoon. If it is only for `slbeta6` change the target in this PR

**Review by abeykoon — APPROVED** [review]

**Review by abeykoon — APPROVED** [review]

File: salesforce/samples/bulk_api_usecases/bulk-operations-csv/bulk_delete_csv.bal [hunk 1]
```diff
@@ -15,7 +15,7 @@
 // under the License.
 
 import ballerina/log;
-import ballerinax/sfdc;
+import ballerinax/salesforce as sfdc;
```
[the remark this answers is not in the document]
Comment (abeykoon) [reply salesforce/samples/bulk_api_usecases/bulk-operations-csv/bulk_delete_csv.bal:18]: @SkNuwanTissera will fix this.

[the remark this answers is not in the document]
Comment (SkNuwanTissera) [reply salesforce/samples/bulk_api_usecases/bulk-operations-csv/bulk_delete_csv.bal:18]: Ack. Updated all samples with changes needed.

File: salesforce/modules/soap/Module.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ Before using this connector in your Ballerina application, complete the followin
...
 ### Step 2: Create a new connector instance
-Create a `sfdc:ConnectionConfig` with the OAuth2 tokens obtained, and initialize the connector with it.
+Create a `salesforce:ConnectionConfig` with the OAuth2 tokens obtained, and initialize the connector with it.
 
 ```ballerina
-sfdc:ConnectionConfig sfConfig = {
+salesforce:ConnectionConfig sfConfig = {
    baseUrl: <"EP_URL">,
    clientConfig: {
      clientId: <"CLIENT_ID">,
```
[the remark this answers is not in the document]
Comment (SkNuwanTissera) [reply salesforce/modules/soap/Module.md:30]: Ack. Updated all samples with changes needed.

**Review by sachinira — APPROVED** [review]

**Pull request merged**